### ISSUE TITLE
[regression-44.0.0 / macOS] Crash on 'activate' event (restoring app by clicking on dock icon)

### ISSUE THREAD

**Issue by ikiK-CRO** [nativefier/nativefier#1212]
I haven't bumped natifefier since 43.something (its been a while.) Just tried new v44.0.0 and on Mac when app window gets closed (not minimized closed, also note I don't use fast quit option for that), app stays "on" with dot next to it in dock but when trying to get back in it there is an error: 
```

A JavaScript error occurred in the main process

Uncaught Exception:
TypeError: Cannot read property 'show' of undefined
at App.<anonymous> (/Applications/Advocatus-darwin-x64 v44.0.0/Advocatus.app/Contents/Resources/app/lib/webpack:/nativefier-placeholder/src/main.ts:177:18)
at App.emit (events.js:315:20)
```

App needs to be shut down and re open to make it work again.

**Comment by ronjouch** [comment]
@ikiK-CRO thx for the fast feedback. I pushed a tentative PR for this, will release a fix shortly.

**Comment by ronjouch** [comment]
Released in Nativefier 44.0.1 , now on npm. Thanks again for the feedback!
